randovania/game_description/integrity_check.py

```diff
@@ -166,7 +166,8 @@ def find_area_errors(game: GameDescription, area: Area) -> Iterator[str]:
             # if this node would satisfy the victory condition, it does not need outgoing connections
             current = ResourceCollection.with_database(game.resource_database)
             current.set_resource(node.event, 1)
-            if game.victory_condition.satisfied(current, 0, game.resource_database):
+            game.create_node_context(current)
+            if game.victory_condition.satisfied(game.create_node_context(current), 0):
```

Comment:
First call to `game.create_node_context(current)` in the L169  looks useless to me

Reply:
Indeed something I forgot to cleanup